test/print_test_stats.py

```diff
@@ -156,26 +183,67 @@ def send_report_to_s3(reports, *, total_seconds):
     # input files of a few megabytes in size like these JSON files, and
     # because for some reason zlib doesn't seem to play nice with the
     # gunzip command whereas Python's bz2 does work with bzip2
-    obj.put(Body=bz2.compress(json.dumps({
-        **build_info(),
-        'total_seconds': total_seconds,
-        'suites': {
-            name: {
-                'total_seconds': suite.total_time,
-                'cases': [
-                    {
-                        'name': case.name,
-                        'seconds': case.time,
-                        'errored': case.errored,
-                        'failed': case.failed,
-                        'skipped': case.skipped,
-                    }
-                    for case in suite.test_cases
-                ],
-            }
-            for name, suite in reports.items()
-        }
-    }).encode()))
+    obj.put(Body=bz2.compress(json.dumps(obj).encode()))
+
+def print_regressions(obj, *, n, stdev_threshold):
```

Comment:
`n` is a vague variable name

Reply:
also nit: the major thing this function does seems like cleaning up the json object to split it into head and base, yes? can we break that out into a separate function --> figure that could be useful elsewhere too.

Reply:
> `n` is a vague variable name

good point, will change that

> also nit: the major thing this function does seems like cleaning up the json object to split it into head and base, yes? can we break that out into a separate function --> figure that could be useful elsewhere too.

not sure exactly what you mean here; the base info isn't in the JSON object it's passed (`obj`), but rather in S3, so it needs to run `git` commands to figure out what commits to grab from S3, then download those, and then pass to `stats_utils`

Reply:
Yes. What I meant was spin off into a `(head, base) = fetch_base_content(obj, n)` 
Then in print function it does fetch, then calls stats_utils and get human-readable format.

Reply:
so you're saying, add a function that computes the 7 arguments to pass to `stats_utils.regression_info`, then call that function, extract the 7 arguments, and pass the results to `stats_utils.regression_info`?

Reply:
nvm. let's do this in follow up PR